src/lib/parseExperience.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file parseExperience.js
+ * Exports a helper method that assists in parsing out a specific experience.
+ */
+
+/**
+ * Helper method that parses an experience, by it's slug, from an experiences array.
+ *
+ * @param {object} experiences
+ *   Object containing experiences.
+ * @param {array} experiences.items
+ *   Array containing experiences.
+ * @param {string} experience.items[].field_experience_path
+ *   String slug of this experience item.
+ * @param {string} experienceSlug
+ *   String specifying the slug of the experience to extract.
+ *
+ * @returns {object|null}
+ *   Experience object, or null.
+ */
+const parseExperience = (experiences = {}, experienceSlug = null) => {
```

Comment:
This is a little helper method that can yank an experience out of an array of experienced based on the slug, but safely return null when the specified experience does not exist.